force-app/main/default/quickActions/Review__c.SubmitReview.quickAction-meta.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<QuickAction xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
@ClayTomerlin , the Quick Action has two pre-defined fields: `Status__c` and `SubmittedDate__c`.   However, both fields are on the Quick Action Layout.   If someone has access to edit the fields on the layout, say an Admin with **Modify All** permission, whatever data that is entered in the Quick Action Layout is committed and the pre-defined formula is ignored.

My recommendation is to not include pre-defined fields on a Quick Action Layout so they are never "overridden" unexpectedly.

I do see the fields are marked read-only on the Layout.   My hunch is only users with **Modify All Records** "Admin" permissions or maybe **Modify All* object-level permission will be able to override the Quick Action Layout's read-only configuration.   I see OFM dev orgs don't include another internal user, e.g. Michon.  Can that be added to our dev org flow?

Also while we are here, why isn't `Comments__c` on the Quick Action Layout?   It seems that's the only input we want to confirm is completed before submitted.

Reply:
@spelak-salesforce These are some of the iterations I went through, actually. For the predefined fields, it will always display the predefined value as the 'default', even in edit mode. However, you are correct that they can only be seen in edit mode when a user has Modify All. I attempted to highlight in the WI chatter updates and PR notes, but I can be a bit more specific in the call out on the WI notes as well.

On`Comments__c`: I originally had it on the layout with the same mindset as you, but decided to remove it after discussing the 'lightweight' goal of keeping 'Submit Review' a clear 'Submit' only process. This also aligns with keeping in mind that the OFM Review processes should be simple and small enough to encourage customization. There will be iterations to add locking record, etc, but this WI is a very simple MVP of what 'Submit Review' means.